modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -382,6 +382,114 @@ definition {
 		}
 	}
 
+	@description = "LPS-165849 - Verify terms from a related custom object (child) are replaced correctly in an email notification"
+	@priority = 5
+	test CanSendEmailNotificationWithTermsComingFromCustomObjectRelationship {
+		property custom.properties = "feature.flag.LPS-165849=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given two custom objects, A and B, each with a text field and publish them") {
+			for (var letters : list "A,B") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = "Custom Object ${letters} 182605",
+					objectName = "CustomObject${letters}182605",
+					pluralLabelName = "Custom Objects ${letters} 182605");
+
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Field Text ${letters}",
+					fieldName = "customField",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "CustomObject${letters}182605");
+
+				ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject${letters}182605");
+			}
+		}
+
+		task ("And given a 1:M relationship between the Object A and Object B") {
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObjectA182605",
+				objectName_2 = "CustomObjectB182605",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+		}
+
+		task ("And given a email notification template using terms from both object definitions") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "[%RELATIONSHIP_CUSTOMOBJECTA182605_CUSTOMFIELD%] | [%CUSTOMOBJECTB182605_CUSTOMFIELD%]",
+				entryTo = "[%CUSTOMOBJECTB182605_AUTHOR_EMAIL_ADDRESS%]",
+				fromAddress = "[%RELATIONSHIP_CUSTOMOBJECTA182605_AUTHOR_EMAIL_ADDRESS%]",
+				fromName = "[%CUSTOMOBJECTB182605_AUTHOR_FIRST_NAME%]",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "[%RELATIONSHIP_CUSTOMOBJECTA182605_AUTHOR_FIRST_NAME%] [%RELATIONSHIP_CUSTOMOBJECTA182605_AUTHOR_LAST_NAME%]");
+
+			PortletEntry.save();
+		}
+
+		task ("And given the title field of custom object A is changed") {
```

Comment:
`And given the title field of Custom Object A is set to the text field`

Keep the capitalization consistent and specify what is the value of the title field